lib/order.rb

```diff
@@ -0,0 +1,52 @@
+require_relative 'menu'
+
+class Order
+  
+  attr_reader :menu, :basket
+
+  def initialize
+  	@menu = Menu.new
+  	@basket = []
+  	@total_amount = 0
+  	@dish = nil
+  	@quantity = 0
+  end
+
+  def add(dish, quantity = 1)
+  	@dish = dish.downcase
+  	@quantity = quantity
+    available_menu ? update_basket : not_update_basket
+  end
+
+  def update_basket
+  	@basket.delete_if { |e| e[:name] == @dish }
+  	@basket << { name: @dish, quantity: @quantity, 
+  	  	        price: menu.list_of_dishes[@dish], 
+  	  	        accumulated_prize: accumulated_prize }
+  end
+
+  def not_update_basket
+  	"We don't have it in the menu."
+  end
```

Comment:
Don't return strings for the user - you do not have an interface, so there is no reason to write messages to the user. If you did have an interface, the responsibility for the message would be in the interface layer, not in the back end.